### PR TITLE
docs: improve blog list page design

### PR DESCRIPTION
**Pull request by jaemig**
This aims to enhance the blog list page by improving the styling of each blog preview item.
Each item is now wrapped by a card and shows more information like the publishing date, the page's description and a badge to highlight that the respective blog entry is new.

### CODE REVIEW

**Comment by schettn** [comment]
It is not possible to click the card, but the hover indicates it.

https://github.com/user-attachments/assets/db076263-de0e-4398-af33-d5c9895f5d54

**Comment by jaemig** [comment]
@schettn Thanks for the feedback. I have adapted the cards accordingly and made additional improvements.

File: docs/pages/blog/pylon-2.0.mdx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
 description: 'Pylon v2 introduces the support for different runtimes, a new create command, and an improved development server.'
+date: 'October 10th, 2024'
+new: true
```
Comment (schettn) [inline docs/pages/blog/pylon-2.0.mdx:4]: New is true but I dont see this "New" flag. Why?

image

File: docs/pages/blog/pylon-1.0.mdx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 description: 'Almost two years of development, countless hours of testing, and a lot of love and passion have gone into the creation of Pylon 1.0. We are excited to share this milestone with you and look forward to seeing how Pylon will revolutionize your GraphQL development experience.'
+date: 'October 9th, 2024'
```
Comment (schettn) [inline docs/pages/blog/pylon-1.0.mdx:3]: Are the dates correct?

File: docs/pages/blog/pylon-2.0.mdx [hunk 3]
```diff
@@ -1,5 +1,7 @@
 ---
 description: 'Pylon v2 introduces the support for different runtimes, a new create command, and an improved development server.'
+date: 'October 10th, 2024'
```
Comment (schettn) [inline docs/pages/blog/pylon-2.0.mdx:3]: Are the dates correct?

**Review by schettn — CHANGES_REQUESTED** [review]

**Review by schettn — APPROVED** [review]

**Pull request merged**